types/draft-js/draft-js-tests.tsx

```diff
@@ -68,13 +68,12 @@ class RichEditorExample extends React.Component<{}, { editorState: EditorState }
     return getDefaultKeyBinding(e);
   }
 
-  handleKeyCommand = (command: string) => {
+  handleKeyCommand = (command: string, editorState: EditorState) => {
```

Comment:
Should `editorState` be optional here?

Reply:
I don't think so, since `editorState` is always passed and as I commented earlier:
If I change this to `editorState?: EditorState`, from a user perspective it looks like that parameter can be `undefined`, which is incorrect.

Reply:
i think that @smvilar is correct - making `editorState` optional would force the consumer to write a `handleKeyCommand` function that accounts for the possibility of an `undefined` `editorState` even though the function should always receive a value..